assets/banCtrl.js

```diff
@@ -1,10 +1,13 @@
 var shared = require('./sharedClientFunctions.js');
 module.exports = function ($scope, io) {
   shared.addRepeats($scope, io);
+  
+  const messageAddition = '\n\n***\n**Please review [this page](https://www.reddit.com/r/pokemontrades/wiki/appeals) for important information before replying or taking any other action.**';
+  
   $scope.banInfo = {
     username: $scope.query.username || '',
     banNote: $scope.query.banNote || '',
-    banMessage: $scope.query.banMessage || '',
+    banMessage: $scope.query.banMessage || messageAddition,
```

Comment:
What if there already is a ban message here? Do we want to append the appeal string?